packages/graphql-auth-transformer/src/__tests__/NonModelAuthTransformer.test.ts

```diff
@@ -1,7 +1,7 @@
 import { GraphQLTransform } from 'graphql-transformer-core';
```

Comment:
These are just some linting cleanups of the V1 transformer test that I applied while I was porting the tests to the V2 transformer